screens/CreateChatScreen.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+    Platform,
+    ScrollView,
+    StyleSheet,
+    TouchableOpacity,
+    View,
+    KeyboardAvoidingView,
+    AsyncStorage
+} from 'react-native';
+import {
+    Container,
+    Header,
+    Title,
+    Content,
+    Form,
+    Label,
+    Button,
+    Left,
+    Right,
+    Body,
+    Icon,
+    Text,
+    List,
+    ListItem,
+    Item,
+    Input
+} from 'native-base';
+import { putChat } from '../actions';
+class CreateChatScreen extends React.Component {
+    static navigationOptions = {
+        header: null
+    };
+
+    state = {
+        name: ''
+    };
+
+    _submitForm = async () => {
+        const { name } = this.state;
+        const storedUser = await AsyncStorage.getItem('user');
```

Comment:
i don't want to do this for every request, so later we should make a helper that will always get the user token before calling the api